src/middlewares/requestId.js

```diff
@@ -5,11 +5,17 @@ let requestId = null;
 /**
  * Express.js middleware
  * responsible for initializing and saving the request id
+ * EDIT: If zipkin is used by client and X-B3-SpanId header are send,
+ * then use X-B3-SpanId as requestid for tracing over multiple apps
  * @return {function} express middleware
  **/
 function middleware() {
 	return (req, res, next) => {
-		requestId = uuidv4();
+		if (Boolean(req.header('X-B3-SpanId'))) {
+			requestId = req.header('X-B3-SpanId');
```

Comment:
You sure that this is something generic that people use ? Or should we just inject a request header as an argument to the middleware method ?